### PR TITLE
rebuild from aws-sdk repo

### PR DESCRIPTION
**Pull request by Benjamin-deVries**
Opened in lieu of: https://github.com/CruGlobal/aws-sdk/pull/8
Upgrades aws-sdk to use the new amazon library.

### CODE REVIEW

**Comment by Benjamin-deVries** [comment]
@Benjamin-deVries needs to update the readme

File: pom.xml [hunk 1]
```diff
@@ -505,24 +505,7 @@
...
-                <scope>provided</scope>
-            </dependency>
-            <!-- Logging Dependencies -->
-            <dependency>
-                <groupId>org.slf4j</groupId>
-                <artifactId>slf4j-api</artifactId>
-                <version>1.7.21</version>
-                <scope>provided</scope>
-            </dependency>
+
```
Comment (wrandall22) [inline pom.xml:559]: The uber-jar dependency, at least, will need to go away, and that one in particular may need to be replaced by the aem sdk dependency.

**Comment by wrandall22** [comment]
This builds and installs locally. It does not make a bundle, so these jars will not be visible to OSGI, which may be fine. I haven't tested commons bringing in aws-sdk as a dependency to see if it will resolve issues. You can try these things in dev, this doesn't seem to break anything.

**Comment by Benjamin-deVries** [comment]
@wrandall22 
Deploying gives these errors:
```
2024-07-22 14:20:21.632 Analyser errors have been found 
2024-07-22 14:20:21.632 The analyser found the following errors for author and publish : 
2024-07-22 14:20:21.632 [api-regions-exportsimports] org.cru:cru-aem-commons.core:3.0.0-SNAPSHOT: Bundle cru-aem-commons.core:3.0.0-SNAPSHOT is importing package(s) [software.amazon.awssdk.auth.credentials, software.amazon.awssdk.http, software.amazon.awssdk.services.ssm, software.amazon.awssdk.regions, software.amazon.awssdk.services.ssm.model, software.amazon.awssdk.awscore.client.builder] in start level 20 but no bundle is exporting these for that start level. (Cru:cru.all:3.0.0-SNAPSHOT)
2024-07-22 14:20:21.632 [api-regions-exportsimports] org.cru:cru-org-cloudsearch.core:3.0.0-SNAPSHOT: Bundle cru-org-cloudsearch.core:3.0.0-SNAPSHOT is importing package(s) [software.amazon.awssdk.auth.credentials, software.amazon.awssdk.core.client.builder, software.amazon.awssdk.services.cloudsearchdomain, software.amazon.awssdk.services.cloudsearchdomain.model, software.amazon.awssdk.core.sync, software.amazon.awssdk.regions, software.amazon.awssdk.awscore.client.builder] in start level 20 but no bundle is exporting these for that start level. (Cru:cru.all:3.0.0-SNAPSHOT)
```

So, looks like it does need to be a bundle to provide these jars to the environment?

**Comment by wrandall22** [comment]
If you are trying to import them through OSGI you will need them to be exposed in OSGI via bundle. If you are depending on them without import, you may be able to get away with not doing that.

**Pull request closed without merging**